sdk/vision/azure-ai-vision-imageanalysis/README.md

```diff
@@ -290,7 +290,31 @@ Message: Status code 400, "{"error":{"code":"InvalidRequest","message":"Image fo
 
 ### Enable HTTP request/response logging
```

Comment:
Nit: May be better to include 'enable client logging' to trouble shooting as well.
Example: https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/containerregistry/azure-containers-containerregistry/TROUBLESHOOTING.md#enable-client-logging

Reply:
Hi @haolingdong-msft thank you for your comment! I looked at the link you suggested. It contains the same information in the link I already include: https://learn.microsoft.com/azure/developer/java/sdk/troubleshooting-overview#use-logging-in-the-azure-sdk-for-java , so I don't see the need to include both. I hope that's okay with you.